fighters/rosetta/src/opff.rs

```diff
@@ -26,6 +83,51 @@ pub fn rosetta_frame_wrapper(fighter: &mut smash::lua2cpp::L2CFighterCommon) {
 
 pub unsafe fn rosetta_frame(fighter: &mut smash::lua2cpp::L2CFighterCommon) {
     if let Some(info) = FrameInfo::update_and_get(fighter) {
-        moveset(&mut *info.boma, info.id, info.cat, info.status_kind, info.situation_kind, info.motion_kind.hash, info.stick_x, info.stick_y, info.facing, info.frame);
+        moveset(fighter, &mut *info.boma, info.id, info.cat, info.status_kind, info.situation_kind, info.motion_kind.hash, info.stick_x, info.stick_y, info.facing, info.frame);
+    }
+}
+#[smashline::weapon_frame_callback]
```

Comment:
Weapons can have their own specific fighter frames, you can change the attribute to this `#[weapon_frame( agent = WEAPON_KIND_ROSETTA_TICO )]`